openwebui/tools/computer_use_tools.py

```
@@ -602,6 +646,7 @@ async def bash_tool(
         __user__: dict = None,
         __files__: Optional[List[dict]] = None,
         __request__=None,
+        __oauth_token__: dict = {},
```

Ruff (0.15.11)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)